### ISSUE TITLE
Safari Live Preview

### ISSUE THREAD

**Issue by bootstraponline** [gollum/gollum#503]
This issue tracks the progress of enabling Safari in Live Preview as mentioned on #496.

- Update Ace
- Add live preview test on Sauce Labs for [OS X Safari](http://sauceio.com/index.php/2012/08/apple-sauce-android-sauce/)

**Comment by tattwamasi** [comment]
Did live preview on Safari become too hard to do, or is it still a possibility?

**Comment by tattwamasi** [comment]
Cool will check it out and try it. Thanks!

**Comment by dometto** [comment]
Sorry, that link was dead. Here is the relevant code: https://github.com/gollum/gollum/blob/master/lib/gollum/app.rb#L58-L62

**Comment by tattwamasi** [comment]
Worked fine on current Safari on current OS X (well, actually on a dev preview beta of 10.11.6, but close enough).   Worked exactly the same as on Chrome from what I saw though when trying things with same page open in both, trying the same things. 

Seems like it'd be ok to enable but all I did was test the buttons and edit and save a page to check it - nothing exhaustive. 

Didn't try it on iOS yet.  Interested to, but based on comments on ACE issues, not expecting anything to really work.

Thanks for the pointer on how to try it.  I appreciate it.
